src/schema.js

```diff
@@ -179,6 +180,35 @@ class Schema {
     });
   }
 
+  mixinNamedParamFacade(api, def) {
```

Comment:
Would be great if these parameters were named the same as where you call the function (`typeDef, def`)